.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
   - docker
 language: node_js
 node_js:
-  - "8.12.0"
+  - "10.13.0"
```

Comment:
Why not just use "10" as it will take the latest LTS version?

Reply:
I think we should keep exact version, like we do in package.json
But we're not consistent, because in Docker image, we download the latest LTS, using `https://deb.nodesource.com/setup_10.x` -> I should change this